### PR TITLE
Fix config file behaviors

### PR DESCRIPTION
**Pull request by mark-dce**

### CODE REVIEW

File: exe/fcrepo_wrapper [hunk 1]
```diff
@@ -8,7 +8,7 @@ options = {}
 OptionParser.new do |opts|
   opts.banner = "Usage: fcrepo_wrapper [options]"
 
-  opts.on("--config", "Load configuration from a file") do |v|
+  opts.on("-cFILE",  "--config FILE", "Load configuration from a file") do |v|
```
Comment (cbeer) [inline exe/fcrepo_wrapper:11]: I think the short-name `-c` is used over in `solr_wrapper` for some other function. I don't know if we should be worried about introducing that kind of inconsistency between the two project.

File: lib/fcrepo_wrapper.rb [hunk 2]
```diff
@@ -22,7 +22,7 @@ def self.default_instance_options=(options)
   end
 
   def self.default_instance(options = {})
-    @default_instance ||= FcrepoWrapper::Instance.new default_instance_options.merge(options)
```
Comment (cbeer) [inline lib/fcrepo_wrapper.rb:25]: It seems like `default_instance_options` might be entirely unused (and that seems to be the case in `solr_wrapper` too..). Can we remove that entirely?

Comment (mark-dce) [inline lib/fcrepo_wrapper.rb:25]: Not quite, the default port is set through default_instance options in [FcrepoWrapper::Configuration.port](https://github.com/cbeer/fcrepo_wrapper/blob/master/lib/fcrepo_wrapper/configuration.rb#L98)

```
    def port
      # Check if the port option has been explicitly set to nil.
      # this means to start fcrepo_wrapper on a random open port
      return nil if options.key?(:port) && !options[:port]
      options[:port] || FcrepoWrapper.default_instance_options[:port]
    end
```

You could factor that out pretty easily and do something like the [.version](https://github.com/cbeer/fcrepo_wrapper/blob/master/lib/fcrepo_wrapper/configuration.rb#L41-L43) default instead:

```
    def version
      @version ||= options.fetch(:version, FcrepoWrapper.default_fcrepo_version)
    end
```

It looks like @cam156 had some reason for [Adding a setter for the default_instance_options](https://github.com/cbeer/solr_wrapper/commit/48ed28f042db253bf1da6889fe46f0e34f570818) in solr_wrapper and it got copied over here.

I'm not sure what you'd ever want to do that, and it took me a fair bit of time to wrap my head around what all was happening, so I'd be happy to take default_instance_options out and follow the pattern for the version defaulting instead.

File: exe/fcrepo_wrapper (hunk 1 above)
Comment (mark-dce) [inline exe/fcrepo_wrapper:11]: Yes `-c` is used for cloud in solr_wrapper.

```
Usage: solr_wrapper [options]
...
        --config                     Load configuration from a file
...
    -c, --cloud                      Run solr in cloud mode
...
```

I don't have strong opinions.  I put `-c` in for consistency with the majority of the other fcrepo_wrapper commands that have short-name versions and hadn't really been thinking about solr_wrapper yet.

I'm good either way.  Let me know which you prefer.

File: lib/fcrepo_wrapper.rb (hunk 2 above)
Comment (carolyncole) [inline lib/fcrepo_wrapper.rb:25]: In solr I wanted to be able to set what port the solr instance runs on. This allows me to  use solrwrapper just like I used to use jettywrapper.

Aka I run configure, and the run start and it hangs around in the background with a development and test instance for me to use.  Check out the use here: https://github.com/psu-stewardship/etda/blob/master/lib/tasks/solr.rake

**Pull request closed without merging**